includes/settings/functions.php

```diff
@@ -1031,23 +1045,26 @@ function transfer_customizer_monetization_settings() {
 
 	// If the setting has already been saved or transferred then simply return
 	// Using 'null' for comparrison becasue if the padlock and support creator messages were unselected they were stored in the database with the value false, but still need to be transferred.
-	if ( get_option( 'coil_monetization_settings_group' ) || ( ! get_option( 'coil_content_settings_posts_group' ) && 'null' === get_theme_mod( 'coil_title_padlock', 'null' ) && 'null' === get_theme_mod( 'coil_show_donation_bar', 'null' ) ) ) {
+	if ( get_option( 'coil_monetization_settings_group' ) && ! get_option( 'coil_content_settings_posts_group' ) && 'null' === get_theme_mod( 'coil_title_padlock', 'null' ) && 'null' === get_theme_mod( 'coil_show_donation_bar', 'null' ) ) {
```

Comment:
was the `||` a bug?

Reply:
I had written this function assuming that if the coil_monetization_settings_group had been created that all the data must already have been copied across from the customizer. However it has since occurred to me that a more comprehensive condition might be safer in case a situation arises where data is somehow saved in both the customizer and the settings panel. This stricter condition ensures that data will always be migrated and removed if it gets stored in the customizer at all. 

However, as I write this I realize that the "get_option( 'coil_monetization_settings_group' )" part of the condition should rather be removed entirely because it serves no purpose. The condition should simply check if there is nothing relevant still stored in the customizer, and if that is true, it should return, regardless of whether settings have or haven't been saved in the settings panel.

I am going to push a new commit updating the condition as follows:
if ( ! get_option( 'coil_content_settings_posts_group' ) && 'null' === get_theme_mod( 'coil_title_padlock', 'null' ) && 'null' === get_theme_mod( 'coil_show_donation_bar', 'null' ) )